pages/data-design/groups.mdx

```diff
@@ -0,0 +1,132 @@
+---
+layout: 'guide'
+title: 'Intro to Group analytics'
+---
+
+import Link from '../../components/Link';
+
+When doing product analytics you may find yourself in a position when you need to analyze behavior of a group of users, or you need to group
+different actions together into a group.
+
+There are two main ways of group analytics that we are going to look at in this article:
+
+- Grouping users
+- Grouping actions
+
+#### Groups concepts
+
+There are two main concepts when working with group analytics. In Avo they are called “Group Types” and “Group Ids”. See reference table below for what “Group Types” and “Group IDs” are called in other platforms.
+
+Group types are the types of groups you manage, for example companies, branches, chats and so on.
+
+Group ids are identifiers of the individual groups of those group types, for example in the “Company” group type you’d have group ids as names of the companies, like “Alibaba”, “Moderna”, “Uber”.
+
+|                      |            |            |
+| -------------------- | ---------- | ---------- |
+| Concepts in Avo      | Group Type | Group Id   |
+| Concept in Amplitude | Group Type | Group Name |
+| Concept in Mixpanel  | Group Key  | Group Id   |
+| Concept in Segment   | Group Id   | Group Name |
+
+## Grouping users
+
+You can associate users with groups. Then that user profile as well as every action user makes will be attributed to the associated group.
+
+This is useful for example for B2B products, when each individual user of the product belongs to a company, and every action of that user should be attributed to that one company.
+
+See further use cases and which analytics platforms support grouping users below in <Link passHref href="/data-design/groups#associate-user-with-group"><a>Associate user with group</a></Link>.
+
+## Grouping actions
```

Comment:
With a recent confusion I saw as the Sales Eng for Axel Springer, was around what Actions are.

I often like talking about user actions (logging user actions rather than logging events), and I believe that was the concept behind naming this section that. But in light that we have a product concept called Actions, I think we should rename this to "Grouping events"

Reply:
```suggestion
## Grouping events
```